draft-jholland-quic-multicast.md

```diff
@@ -230,14 +232,16 @@ Since clients can join later than a channel began, it is RECOMMENDED that client
 Clients should therefore begin with a high initial_max_streams_uni or send an early MAX_STREAMS type 0x13 value (see Section 19.11 of {{RFC9000}}) with a high limit.
 Clients MAY use the maximum 2^60 for this high initial limit, but the specific choice is implementation-dependent.
 
+The same stream ID may be used in both one or more multicast channels and the unicast connection. Streams behave exactly as specified in Section 2.2 of {{RFC9000}}. However, clients need to be able to handle cases where they receive the same stream data on multiple channels.
```

Comment:
Better, but there's still something here bothering me.

How about:

~~~
Stream frames for server-initiated streams can be sent on multicast
channels as well as the unicast connection.  Doing so is only possible
with stream data that's identical for the same offset and stream ID
across all receivers on that channel.   Stream data for the same offset
might also be sent on the unicast channel or on other multicast
channels.  As described in Section 2.2 of {{RFC9000}}, stream data
received multiple times for the same offset MUST be identical;
if it's not identical it MAY be treated as a connection error of type
PROTOCOL_VIOLATION.
~~~

Maybe that's too wordy, not sure.  The main issues I'm trying to make different in this text:
- I think it's wrong to say it's exactly as in section 2.2 of 9000 because we do differ on flow control, which I think we cover in the next section but also has a MUST in 2.2 of 9000.
- The "however" sentence seems strange because handling it when you get the same stream data is already required in 9000 (but is phrased as "received multiple times" instead of "received on multiple channels").